nntrainer/models/neuralnet.h

```diff
@@ -518,6 +482,17 @@ class NeuralNetwork : public ml::train::Model {
   void disableDynamicFineTuning() { dynamic_training_opt.disable(); }
 
 private:
+  using FlexiblePropTypes =
+    std::tuple<props::Epochs, props::BatchSize, props::SavePath>;
+  using RigidPropTypes = std::tuple<props::LossType>;
+
+  RigidPropTypes model_props;         /**< model props */
+  FlexiblePropTypes model_flex_props; /**< model train props */
+  props::ContinueTrain
+    continue_train; /**< true if epochs and iteration should be retained during
+                       separate model run */
+  std::string load_path; /**< path to load weights when initialize  */
```

Comment:
load_path and save_path are different?

Reply:
load_path is from load(INI_WITH_BIN)

save_path can change later by properties, but should not override load_path.

Reply:
ok.

Reply:
This won't be property and file check is being done at opening so I guess string fits here.